bindata/cloud-network-config-controller/managed/controller.yaml

```diff
@@ -227,6 +230,12 @@ spec:
           items:
             - key: ca.crt
               path: ca.crt
+      - name: trusted-ca
+        configMap:
+          name: trusted-ca-bundle-managed
```

Comment:
this ConfigMap is maanged by Hypershift:

https://github.com/openshift/hypershift/blob/1f9c369e5905137cbeccd452e4ba94c406657bd2/control-plane-operator/controllers/hostedcontrolplane/manifests/pki.go#L89